packages/components/image/src/index.vue

```diff
@@ -130,6 +130,18 @@ export default defineComponent({
     })
 
     const preview = computed(() => {
+      document.body.addEventListener('wheel', e => {
+        if (e.ctrlKey) {
+          if (e.deltaY < 0) {
+            e.preventDefault()
+            return false
+          }
+          if (e.deltaY > 0) {
+            e.preventDefault()
+            return false
+          }
+        }
+      }, { passive: false })
```

Comment:
Humm,
1. I don't think binding event in `computed` is a good idea
2. plus you didn't remove the listener since you are attaching this event to the `body` element

Reply:
ok, i will change it from computed the `chlickHandler()`, and remove the listenenr in `closeViewer()`

Reply:
@JeremyWuuuuu is this ok now